ui/round/src/ctrl.ts

```diff
@@ -570,6 +572,9 @@ export default class RoundController implements MoveRootCtrl {
         this.opts.chat?.instance?.post('Good game, well played');
     }
     endGameView();
+    if (window.matchMedia('(max-width: 800px) and (orientation: portrait)').matches) {
```

Comment:
one last nitpick, i believe there is a function for this in common/device called isCol1.